test/config/k8s/test-env.sh.yml

```diff
@@ -74,10 +75,6 @@ spec:
           - name: CONJUR_ACCOUNT
             value: ${CONJUR_ACCOUNT}
 
-          - name: CONJUR_AUTHN_LOGIN
-            value: >-
-              host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${TEST_APP_NAMESPACE_NAME}/*/*
-
```

Comment:
In our docs, we ask that the customer provide this `CONJUR_AUTHN_LOGIN` variable in this manifest. For the sake of uniformity, can we find a way to keep `CONJUR_AUTHN_LOGIN` here and dynamically provide it's value before the tests run. We run the deployment in `deploy_test_env` and since you are already exported, before running the actual test so supply it. Like so:
```
- name: CONJUR_AUTHN_LOGIN
   value: ${CONJUR_AUTHN_LOGIN_KEY_VALUE}
```

Reply:
that's not how `CONJUR_AUTHN_LOGIN_KEY_VALUE` works. please note that we do the same with `SECRETS_DESTINATION`, `CONTAINER_MODE` & `K8S_SECRETS`. All these variables are in the docs and will also be in the manifest that will be created once the bash script finishes to add the variables. 

please go over this method and tell me if you better understand it: https://github.com/cyberark/secrets-provider-for-k8s/blob/master/test/utils.sh#L202

if you still think we should change the way we add variables dynamically tell me.